playbooks/scenarios/test_cases/write_clone_entry.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019 Cable Television Laboratories, Inc.
```

Comment:
Please remove clear_clone_entry.yml and just use write_clone_entry and set the var.clone from core/data-inspection.yml